extensions/ql-vscode/src/language-support/ast-viewer/ast-viewer.ts

```diff
@@ -80,11 +80,11 @@ class AstViewerDataProvider
     const treeItem = new TreeItem(item.label || "", state);
     treeItem.description = line ? `Line ${line}` : "";
     treeItem.id = String(item.id);
-    treeItem.tooltip = `${treeItem.description} ${treeItem.label}`;
+    treeItem.tooltip = `${treeItem.description} ${typeof treeItem.label === "string" ? treeItem.label : (treeItem.label?.label ?? "")}`;
```

Comment:
This is a bit ugly but the type of `treeItem.label` is `string | TreeItemLabel | undefined` and it doesn't seem that `TreeItemLabel` implements `toString()`.